packages/mantine/src/components/copyToClipboard/CopyToClipboard.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+
+import {CheckSize24Px, CopySize24Px} from '@coveord/plasma-react-icons';
+import {TextInput, CopyButton, Tooltip, ActionIcon, createStyles, Textarea} from '@mantine/core';
+
+export enum CopyToClipboardVariant {
+    Button = 'button',
+    Text = 'text',
+    TextArea = 'textarea',
+}
```

Comment:
No need for an enum. Since the variant prop is already typed using a union of strings, it already provides type protection and autocompletion.

```suggestion
```